doc/dev_guide/release_management.rst

```diff
@@ -10,92 +10,134 @@ Release management
 Release policy
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-A Tarantool release is identified by three digits, for example, 1.7.7.
-We use these digits according to their definitions provided at http://semver.org:
+A Tarantool release is identified by three digits, for example, 1.10.7:
+
+* The first digit stands for a MAJOR release that introduces
+  some *major changes*. Up to now, there has been only one major release jump
+  when we delivered the 2.x release series with the SQL support.
+* The second digit stands for a MINOR release that is used for
+  introducing new *features*. Some of the features may have no backward
+  compatibility.
+* The third digit is for PATCH releases that contain only
+  backward-compatible *bug fixes*.
```

Comment:
It is right by meaning, but technically it is true since a first beta release within a series. An alpha (`x.y.0`) is not a release in fact, it is more like a point from where a series starts to live. Look at the diagram:

```
2.2 series ---- 2.2.1 ---- 2.2.2 ---- 2.2.3
                  |   
                  V   
2.3 series .... 2.3.0 ---- 2.3.1 ---- 2.3.2 ---- 2.3.3
                             |   
                             V   
2.4 series ............... 2.4.0 ---- 2.4.1 ---- 2.4.2
                                        |   
                                        V   
2.5 series .......................... 2.5.0 ---- 2.5.1
                                                   |   
                                                   V   
2.6 series ..................................... 2.6.0

--------------------- time axis --------------------->
```

`x.y.0` → `x.y.1` is where features are land and backward incompatible changes are possible. After `x.y.1` only backward-compatible bugfixes are land.

Another interesting thing is that `x.y.0` is equivalent of `x.(y-1).1` in practice.

Reply:
NB: If we'll place the diagram into the documentation, it worth to add some terminator symbol after `x.y.3`.